libs/vertexai/langchain_google_vertexai/_base.py

```diff
@@ -215,6 +215,11 @@ class _VertexAICommon(_VertexAIBase):
     model_name will be used to determine the model family
     """
 
+    cached_content: Optional[str] = None
```

Comment:
I'm sorry, but how would it be different from https://github.com/langchain-ai/langchain-google/blob/8c569edcab6b48f3f57390ded48f1af2e75eac8b/libs/vertexai/langchain_google_vertexai/chat_models.py#L1045?